### DISCUSSION TITLE
Meaning of srv and nsu in an nodeid

### DISCUSSION THREAD

**Discussion by swamper123** [FreeOpcUa/opcua-asyncio#442] [Q&A]
Hello there,

I am working on the generate_*.py files and tumbled over that snippit here:

https://github.com/FreeOpcUa/opcua-asyncio/blob/fa043d890111ea35b92b8d751d3da87c344b8c6b/schemas/generate_address_space.py#L79

I know that `nsu=` shall be used, if the server namespace is Null or empty, but it isn't used in the code (I guess namespace should be `nsu` then?).

But what shall srv mean? I just found https://reference.opcfoundation.org/v104/GDS/docs/C.1/ but I guess this isn't meant.

**Comment by AndreasHeine** [comment]
https://reference.opcfoundation.org/v104/Core/docs/Part4/7.11/
